### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by badger-99**
This pull request will introduce a JavaScript-enabled pop-up menu for the mobile version of the webpage.

### CODE REVIEW

**Review by amejid — COMMENTED** [review]
Hi team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project, that does not have ESLint checks running. Please use the `linters.yml` file from the [HTML-CSS-JS](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) folder of the Linters config repository.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -15,16 +15,16 @@
   <!-- Header -->
   <header id="toolbar">
     <div id="logo">
-      <a class='link' href="">Alfred</a>
+      <a class='link' href="#headline">Alfred</a>
     </div>
```
Comment (Shisui6) [inline index.html:19]: Currently, an element in your page is affecting your mobile version and mobile menu design on smaller screens. (< 680px). This is causing a lot of horizontal whitespace and not allowing your mobile menu to be fullscreen like the figma design. Please look into your mobile styling and ensure that there is no horizontal space and the mobile menu is full screen. Below I have attached screenshots for 375px widths:

Mobile version:
image

Mobile menu:
image

Figma design:
image

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @badger-99, 

# Requires Changes 🍀
You have done a great job so far!👏👏
You still need to work on a few issues before moving on to the next project, but you're almost there🪲!

## Highlights🎯
- Mobile menu functionality fully implemented ✅


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🪶
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @badger-99 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Designed mobile overlay according to the template ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**